packages/portal/spec-renderer/src/components/SpecDetails.vue

```diff
@@ -80,8 +80,15 @@ const showAndScrollToOperation = (): void => {
     return
   }
 
-  swaggerRef.value.showOperation(props.activeOperation)
-  swaggerRef.value.scrollToOperation(props.activeOperation)
+  const activeOperation = { ...props.activeOperation }
+
+  if (activeOperation && activeOperation.tag) {
+    const newStr: string = decodeURIComponent(activeOperation.tag.trim().replaceAll(/\s/g, '%20'))
```

Comment:
can you just add a note that this is the same logic from the other repo? I know they're tightly coupled.